data_steward/analytics/tools/email_generator/dictionaries_and_lists.py

```diff
@@ -0,0 +1,335 @@
+
+    # general concept success rate columns
+    'observation_success_rate': 'Observation',
+    'drug_success_rate': 'Drug Exposure',
+    'procedure_success_rate': 'Procedure Occurrence',
+    'condition_success_rate': 'Condition Occurrence',
+    'measurement_success_rate': 'Measurement',
+    'visit_success_rate': 'Visit Occurrence',
+
+    # field concept success rate columns
+    'total_unit_success_rate': 'Measurement',
+    'total_route_success_rate': 'Drug Exposure',
+
+    # drug integration columns
+    'all_drugs': 'All Drugs',
+    'ace_inhibitors': 'ACE Inhibitors',
+    'painnsaids': 'Pain NSAIDS',
+    'msknsaids': 'MSK NSAIDS',
+    'statins': 'Statins',
+    'antibiotics': 'Antibiotics',
+    'opioids': 'Opioids',
+    'oralhypoglycemics': 'Oral Hypoglycemics',
+    'vaccine': 'Vaccine',
+    'ccb': 'Calcium Channel Blockers',
+    'diuretics': 'Diuretics',
+
+    # measurement integration columns
+    'All_Measurements': 'All Measurements',
+    "Physical_Measurement": 'Physical Measurements',
+    'CMP': 'Comprehensive Metabolic Panel',
+    'CBCwDiff': 'CBC with Differential',
+    'CBC': 'Complete Blood Count (CBC)',
+    'Lipid': 'Lipid'}
+
+data_quality_dimension_dict = {
+    'concept': 'Conformance',
+    'duplicates': 'Plausibility',
+    'end_before_begin': 'Plausibility',
+    'data_after_death': 'Plausibility',
+    'sites_measurement': 'Completeness',
+    'drug_success': 'Completeness',
+    'drug_routes': 'Completeness',
+    'measurement_units': 'Completeness',
+    'date_datetime_disparity': 'Conformance',
+    'erroneous_dates': 'Plausibility',
+    'person_id_failure_rate': 'Conformance'}
+
+metric_type_to_english_dict = {
+    # field population metrics
+    'measurement_units': 'Unit Concept ID Success Rate',
+    'drug_routes': 'Route Concept ID Success Rate',
+
+    # integration metrics
+    'drug_success': 'Drug Ingredient Integration',
+    'sites_measurement': 'Measurement Integration',
+
+    # ACHILLES errors
+    'end_before_begin': 'End Dates Preceding Start Dates',
+    'data_after_death': 'Data After Death',
+    'date_datetime_disparity': 'Date/Datetime Disparity',
+
+    # other metrics
+    'concept': 'Concept ID Success Rate',
+    'duplicates': 'Duplicate Records',
+    'erroneous_dates': 'Erroneous Dates',
+    'person_id_failure_rate': 'Person ID Failure Rate'}
+
+full_names = {
+    "saou_uab_selma": "UAB Selma",
+    "saou_uab_hunt": "UAB Huntsville",
+    "saou_tul": "Tulane University",
+    "pitt_temple": "Temple University",
+    "saou_lsu": "Louisiana State University",
+    "trans_am_meyers": "Reliant Medical Group (Meyers Primary Care)",
+    "trans_am_essentia": "Essentia Health Superior Clinic",
+    "saou_ummc": "University of Mississippi",
+    "seec_miami": "SouthEast Enrollment Center Miami",
+    "seec_morehouse": "SouthEast Enrollment Center Morehouse",
+    "seec_emory": "SouthEast Enrollment Center Emory",
+    "uamc_banner": "Banner Health",
```

Comment:
These exist in the table `lookup_tables.hpo_site_id_mappings`. If this PR is focused on improvements, I'd suggest using that table

Reply:
I'll put this in the EDQ-440 ticket to polish up scripts later on. These won't be major changes but figured we should tackle the time-sensitive stuff first.